### ISSUE TITLE
Error on pytex watch

### ISSUE THREAD

**Issue by wichtounet** [GaretJax/pytex#1]
Hi, 

I've got an error when I try to do a pytex watch in my report directory: 

```
  INFO: Watching /home/wichtounet/Dropbox/Perso/School/Master/project_pa/report for changes...
 ERROR: Exception in thread Thread-1:
 ERROR: Traceback (most recent call last):
 ERROR:   File "/usr/lib64/python2.7/threading.py", line 551, in __bootstrap_inner
 ERROR:     self.run()
 ERROR:   File "/usr/lib64/python2.7/site-packages/pyinotify-0.9.3-py2.7.egg/pyinotify.py", line 1513, in run
 ERROR:     self.loop()
 ERROR:   File "/usr/lib64/python2.7/site-packages/pyinotify-0.9.3-py2.7.egg/pyinotify.py", line 1499, in loop
 ERROR:     self.process_events()
 ERROR:   File "/usr/lib64/python2.7/site-packages/pyinotify-0.9.3-py2.7.egg/pyinotify.py", line 1295, in process_events
 ERROR:     watch_.proc_fun(revent)  # user processings
 ERROR:   File "/usr/lib64/python2.7/site-packages/pytex-0.1a-py2.7.egg/pytex/monitors/pyinotify.py", line 39, in handle_event
 ERROR:     self.final_callback(self.EVENT_MAPPINGS[event.mask](event.name))
 ERROR: KeyError: 1
 ERROR: 
```

Do you have an idea of where does that error comes from ?

Thanks

**Comment by GaretJax** [comment]
Thanks for the report.

In fact I didn't test the inofity wrapper that much (I use fsevents on OS X). The inotify support on OS X does not send `IN_ACCESS` events and as such I didn't add it (as well as of couple of others) to my callback mappings (see `pytex/monitors/pyinotify.py:11`)

I'll add a fallback as soon as possible!

**Comment by GaretJax** [comment]
Could you check the latest commit on the `develop` branch to check that everything is working properly, pls?

**Comment by wichtounet** [comment]
It is a bit better, there are no error left, but once I do a modification to a file, the recompilation is done infinitely.

**Comment by GaretJax** [comment]
Which files are triggering the recompilation?

**Comment by wichtounet** [comment]
Whatever files is changed, the recompilation is done forever. I tried to change the master.tex file, a sub file included in master or ever touch a new tex file. The results are the same.

**Comment by wichtounet** [comment]
I tried again just now and a new error just arrived: 

```
 ERROR: Exception in thread Thread-1:
 ERROR: Traceback (most recent call last):
 ERROR:   File "/usr/lib64/python2.7/threading.py", line 551, in __bootstrap_inner
 ERROR:     self.run()
 ERROR:   File "/usr/lib64/python2.7/site-packages/pyinotify-0.9.3-py2.7.egg/pyinotify.py", line 1513, in run
 ERROR:     self.loop()
 ERROR:   File "/usr/lib64/python2.7/site-packages/pyinotify-0.9.3-py2.7.egg/pyinotify.py", line 1499, in loop
 ERROR:     self.process_events()
 ERROR:   File "/usr/lib64/python2.7/site-packages/pyinotify-0.9.3-py2.7.egg/pyinotify.py", line 1295, in process_events
 ERROR:     watch_.proc_fun(revent)  # user processings
 ERROR:   File "/usr/lib64/python2.7/site-packages/pytex-0.1a-py2.7.egg/pytex/monitors/pyinotify.py", line 43, in handle_event
 ERROR:     self.final_callback(self.EVENT_MAPPINGS[key](*paths))
 ERROR:   File "/usr/lib64/python2.7/site-packages/pytex-0.1a-py2.7.egg/pytex/commands/building.py", line 108, in handler
 ERROR:     relative = event.path[len(base) + 1:]
 ERROR: AttributeError: 'FileMoved' object has no attribute 'path'
 ERROR: 
```

But I didnt move any file. Just edited the master.tex file.

**Comment by GaretJax** [comment]
Fixed in 2d4e773d
